app/models/concerns/post_concerns/autoflagging.rb

```diff
@@ -167,8 +170,11 @@ def send_not_autoflagged
     def eligible_flaggers
       conditions = site.flag_conditions.where(flags_enabled: true)
       available_user_ids = {}
+      weight = reasons.pluck(:weight).reduce(:+)
+      poster_rep = stack_exchange_user.reputation
+      reasons_count = reasons.count
       conditions.each do |condition|
-        if condition.validate!(self)
+        if condition.validate_weight_rep_reason_count!(weight, poster_rep, reasons_count)
```

Comment:
Same as above, `condition.validate_post! self` is an available option.

Reply:
Please see my prior comment.